iml-agent/src/daemon_plugins/postoffice.rs

```diff
@@ -24,21 +24,17 @@ use stream_cancel::{Trigger, Tripwire};
 use tokio::{fs, net::UnixListener, sync::Mutex};
 use tokio_util::codec::{BytesCodec, FramedRead};
 
-pub struct POWD(pub Option<WatchDescriptor>);
```

Comment:
Is this not needed?

Reply:
No, not needed. I rely on the kernel not close all watches when the main handle is closed.